src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts

```diff
@@ -430,12 +430,13 @@ export class JJCountComponent implements OnInit, OnChanges {
     // surcharge is always 15%
     if (eventValue == "yes") {
       this.form.get('totalFineAmount').setValue(Math.round(this.form.get('lesserOrGreaterAmount').value));
-      this.lesserOrGreaterAmount = Math.round(this.form.get('lesserOrGreaterAmount').value / 1.15);
-      this.surcharge = Math.round(0.15 * this.lesserOrGreaterAmount);
+      this.lesserOrGreaterAmount = this.form.get('lesserOrGreaterAmount').value / 1.15;
+      this.surcharge = 0.15 * this.lesserOrGreaterAmount;

```

Comment:
I understand there was having problem of round up/down but the one in a few lines later do round up/down. Do we need to align the rounding for surcharge?

Reply:
Since we're calculating the surcharge separately and then rounding only the surcharge and adding to the lesserOrGreaterAmountValue now a few lines below, it's always an accurate calculation for the newTotalFineAmount. The problem above was rounding the whole calculation which eventually didn't match the carried over total fine amount in some cases. So, I think this calculation should always be accurate.